pom.xml

```diff
@@ -80,6 +80,20 @@
             <version>5.10.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.7.0</version>
```

Comment:
Ich fände es besser, wenn wir das in eine Variable packen

<mockito.version>5.7.0</mockito.version>